eclipse-sw360/sw360
# fix(api): Deleting a project returns status code 500 and the API doc for linking a release to a release makes an ambiguous

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

Refer issue: #1876

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
1. Delete a project
Step 1: Admin user creates a project
Step 2: Delete operation is performed by another user who does not have written permission to the created record (Endpoint: /projects/{id}; Method: DELETE)
Expected: Returns a status code 202. Sign in as an Admin user and confirm that the corresponding Moderation Request is created on the Request tab

2. Link release to release
- Just update the releaseID in the example of API doc.

### Checklist
Must:
